docs/configuration.md

```diff
@@ -0,0 +1,49 @@
+## Configuration
+This bot can pull configuration from one of multiple locations. Each parameter will be parsed independently in the following order of precedence:
+1. CLI arguments
+    - See help menu by running `poglink -h` for more information.
+2. Configuration File
+    - Assumed to be named `config.yaml` within the data directory.
+    - Data directory is set to `~/.poglink` unless otherwise specified.
```

Comment:
```suggestion
    - Data directory is set to `~/.poglink` when running natively unless otherwise specified. See [Docker Instructions](https://github.com/FM-17/poglink/blob/main/docs/docker-installation.md) for more informations on how to configure this directory as a bind-mount.
    - See the supplied [sample configuration file](https://github.com/FM-17/poglink/blob/main/sample-config.yaml) for a reference of how to set parameters in this way.
```
The docker image assumes /data will be used.